app/jobs/data_warehouse/table_summary_stats_export_job.rb

```diff
@@ -5,6 +5,8 @@ class TableSummaryStatsExportJob < BaseJob
     REPORT_NAME = 'table_summary_stats'
 
     def perform(timestamp)
+      return if !IdentityConfig.store.data_warehouse_enabled
```

Comment:
let's add some spec coverage for this?

Reply:
already had it coming, forgot to commit/push should be in here now